### PR TITLE
feat(titus): Mimicker property testing

### PR DESCRIPTION
**Pull request by robzienert**
I wanted to have a concrete example to show for the https://github.com/spinnaker/kork/pull/373 work. Surprisingly, there weren't any changes I needed to make as a result. The tests do likely need to get more coverage regardless.

### CODE REVIEW

File: clouddriver-titus/src/test/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/actions/PrepareTitusDeployActionSpec.groovy [hunk 1]
```diff
@@ -126,24 +168,34 @@ class PrepareTitusDeployActionSpec extends Specification {
...
 
-    description.securityGroups == ["sg-2", "sg-1"]
+    description.securityGroups == [sg2Id, sg1Id]
   }
 
   @Unroll
   def "security groups include app security group (label=#labelValue, desc=#descriptionValue, includesAppGroup=#includesAppGroup)"() {
     given:
-    TitusDeployDescription description = createTitusDeployDescription()
+    String sg1Id = mimicker.aws().securityGroupId
```
Comment (cfieber) [inline clouddriver-titus/src/test/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/actions/PrepareTitusDeployActionSpec.groovy]: as a mild bit of test coding style I'd generally prefer to stick these kinds of test data in the `where:` section. I suspect that @robfletcher would have the most valid opinion as senior VP of spock testing though

Comment (robzienert) [reply clouddriver-titus/src/test/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/actions/PrepareTitusDeployActionSpec.groovy]: Oh, sure yeah that makes sense... I'll change.

File: gradle.properties [hunk 2]
```diff
@@ -2,6 +2,6 @@
 includeCloudProviders=all
 fiatVersion=1.4.0
 enablePublishing=false
-korkVersion=6.6.2
+korkVersion=6.5.0-SNAPSHOT
```
Comment (emjburns) [inline gradle.properties]: should this be the snapshot version, or is this leftover from testing locally?

File: clouddriver-titus/src/test/resources/mimicker-titus.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+mimicker:
+  titus:
```
Comment (emjburns) [inline clouddriver-titus/src/test/resources/mimicker-titus.yml]: does this give you a place to define some inputs or configure the behavior for the instance of mimicker that you create?

File: gradle.properties (hunk 2 above)
Comment (emjburns) [reply gradle.properties]: oh I see the PR wasn't merged into kork yet.

File: clouddriver-titus/src/test/resources/mimicker-titus.yml (hunk 3 above)
Comment (robzienert) [reply clouddriver-titus/src/test/resources/mimicker-titus.yml]: It gives you a place to define inputs. Example:

```yaml
mimicker:
  titus:
    cells:
    - cell1
    - cell2
    - cell3
```

```kotlin
dataContainer.list<String>("titus/cells") // returns full list of titus cells.
```

**Pull request merged**